src/logging/ILogger.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LogLevel} from './LoggingService';
+
+/**
+ * The Hazelcast logging interface.
+ */
+export interface ILogger {
+
+    /**
+     * Logs a message with an associated objectName and furtherInfo at the given level.
+     * @param {LogLevel} the log level
```

Comment:
Is this the correct format for TSDoc? I think the first argument after `@param` should be the name of the argument to be explained. Can you check and fix the TSDoc format in other methods too?